githubapi_class.inc

```diff
@@ -101,6 +101,7 @@ class GitHubAPI {
     $this->curl->setHeader('Content-Type', 'application/json');
     $this->curl->setUserAgent('Backdrop CMS GitHub API module');
     $this->curl->setHeader('Accept', '*/*');
+    $this->curl->setHeader('Authorization', 'token' . $this->getToken());
```

Comment:
Looks like there should be a space after the word "token"? https://developer.github.com/v3/auth/#authenticating-for-saml-sso Maybe it's not necessary, but the API samples include it.